wsmaster/che-core-api-workspace/src/test/resources/devfile/schema_test/command/devfile_command_with_preview_url_without_params.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+---
+apiVersion: 1.0.1-beta
+metadata:
+  name: petclinic-dev-environment
+components:
+  - alias: mysql
+    type: openshift
+    reference: petclinic.yaml
+    selector:
+      app.kubernetes.io/name: mysql
+      app.kubernetes.io/component: database
+      app.kubernetes.io/part-of: petclinic
+commands:
+  - name: build
+    previewUrl:
```

Comment:
it seems to be empty, --  maybe align filename meeded ?

Reply:
filename changed to `devfile_command_with_empty_preview_url.yaml`